---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Drosophila Brahma complex remodels nucleosome organizations in multiple aspects
subtitle: ''
summary: ''
authors:
- Jiejun Shi
- Meizhu Zheng
- Youqiong Ye
- Min Li
- Xiaolong Chen
- Xinjie Hu
- Jin Sun
- Xiaobai Zhang
- Cizhong Jiang
author_notes:
- "Equal contribution"
- "Equal contribution"
tags:
- AT Rich Sequence
- Binding Sites
- Cell Cycle Proteins
- Chromatin Assembly and Disassembly
- Drosophila melanogaster
- Drosophila Proteins
- Nucleosomes
- Nucleotide Motifs
- Trans-Activators
- Transcription Factors
categories: []
date: '2014-09-01'
lastmod: 2022-01-02T17:57:55+08:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-01-02T09:57:55.412623Z'
publication_types:
- '2'
abstract: ATP-dependent chromatin remodeling complexes regulate nucleosome organizations.
  In Drosophila, gene Brm encodes the core Brahma complex, the ATPase subunit of SWI/SNF
  class of chromatin remodelers. Its role in modulating the nucleosome landscape in
  vivo is unclear. In this study, we knocked down Brm in Drosophila third instar larvae
  to explore the changes in nucleosome profiles and global gene transcription. The
  results show that Brm knockdown leads to nucleosome occupancy changes throughout
  the entire genome with a bias in occupancy decrease. In contrast, the knockdown
  has limited impacts on nucleosome position shift. The knockdown also alters another
  important physical property of nucleosome positioning, fuzziness. Nucleosome position
  shift, gain or loss and fuzziness changes are all enriched in promoter regions.
  Nucleosome arrays around the 5' ends of genes are reorganized in five patterns as
  a result of Brm knockdown. Intriguingly, the concomitant changes in the genes adjacent
  to the Brahma-dependent remodeling regions have important roles in development and
  morphogenesis. Further analyses reveal abundance of AT-rich motifs for transcription
  factors in the remodeling regions.
publication: '*Nucleic Acids Research*'
doi: 10.1093/nar/gku717
---
