---
# Documentation: https://wowchemy.com/docs/managing-content/

title: The concurrence of DNA methylation and demethylation is associated with transcription
  regulation
subtitle: ''
summary: ''
authors:
- Jiejun Shi
- Jianfeng Xu
- Yiling Elaine Chen
- Jason Sheng Li
- Ya Cui
- Lanlan Shen
- Jingyi Jessica Li
- Wei Li
tags:
- Animals
- Chromatin
- CpG Islands
- DNA
- DNA Methylation
- DNA Modification Methylases
- DNA-Binding Proteins
- Epigenesis
- Genetic
- Gene Ontology
- Genome
- Histones
- Humans
- Isoenzymes
- Mice
- Molecular Sequence Annotation
- Mouse Embryonic Stem Cells
- Neoplasms
- Proto-Oncogene Proteins
- T-Lymphocytes
- Transcription
- Genetic
categories: []
date: '2021-09-01'
lastmod: 2022-01-02T17:57:51+08:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-01-02T09:57:51.337971Z'
publication_types:
- '2'
abstract: The mammalian DNA methylome is formed by two antagonizing processes, methylation
  by DNA methyltransferases (DNMT) and demethylation by ten-eleven translocation (TET)
  dioxygenases. Although the dynamics of either methylation or demethylation have
  been intensively studied in the past decade, the direct effects of their interaction
  on gene expression remain elusive. Here, we quantify the concurrence of DNA methylation
  and demethylation by the percentage of unmethylated CpGs within a partially methylated
  read from bisulfite sequencing. After verifying 'methylation concurrence' by its
  strong association with the co-localization of DNMT and TET enzymes, we observe
  that methylation concurrence is strongly correlated with gene expression. Notably,
  elevated methylation concurrence in tumors is associated with the repression of
  40~60% of tumor suppressor genes, which cannot be explained by promoter hypermethylation
  alone. Furthermore, methylation concurrence can be used to stratify large undermethylated
  regions with negligible differences in average methylation into two subgroups with
  distinct chromatin accessibility and gene regulation patterns. Together, methylation
  concurrence represents a unique methylation metric important for transcription regulation
  and is distinct from conventional metrics, such as average methylation and methylation
  variation.
publication: '*Nature Communications*'
doi: 10.1038/s41467-021-25521-7
links:
- name: URL
  url: https://www.nature.com/articles/s41467-021-25521-7
---
